specification/graphrbac/data-plane/Microsoft.GraphRbac/stable/1.6/graphrbac.json

```diff
@@ -3191,6 +3225,20 @@
       },
       "description": "Application list operation result."
     },
+    "ServicePrincipalObjectResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "string",
+          "description": "Service Principal Application Object Id"
+        },
```

Comment:
isnt value an array of strings?
{
  "odata.metadata": "https://graph.windows.net/myorganization/$metadata#Edm.String",
  "value": [
    "00b4e797-7017-4720-b187-b01981c820d6"
  ]
}